features/multi_client.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Initializing nultiple clients sends payloads with correct API key to each
+
+Scenario: Unhandled error reported by two clients with different API keys
```

Comment:
I initially didn't understand the expectations of this scenario, since there are two cases rolled into one. Perhaps the name could be amended or additional feature description added to describe the test? Any non-"Scenario" lines leading up to each scenario is treated as a comment and printed on the console during the test run.

As I understand it, the intended effect is:
1. Two clients are configured
2. The device is offline
3. An unhandled error occurs, and a report is cached on disk
4. The app is restarted with delivery enabled
5. One request is sent by each client, each using the correct API key

If that's correct, I'd rename the scenario to something like:

> An unhandled error captured while offline is detected by two clients with different API keys

And maybe add a bit of preamble:

```gherkin
Feature: Multi-client support

    Multiple Bugsnag clients can be configured, each with its own API key and
    configuration options. A report which is captured by a given client should
    use the correct key in report.

    Scenario: An unhandled error captured while offline is detected by two clients with different API keys
```

It also surfaces what additional scenarios should be added (two clients sending unhandled errors while online? two clients sending handled errors offline? etc)